.github/workflows/quality.yaml

```diff
@@ -54,6 +54,9 @@ jobs:
       - name: Clippy (mshv)
         run: cargo clippy --all --all-targets --no-default-features --tests --features "mshv" -- -D warnings
 
+      - name: Clippy (default + amx)
+        run: cargo clippy --all --all-targets --no-default-features --tests --features "amx" -- -D warnings
```

Comment:
You definitely don't want `--no-default-features" here.

Reply:
Oh goodness, I copied it then changed it to run locally and completely forgot X(.